packages/shared/src/components/profile/ProfileSettingsMenu.tsx

```diff
@@ -202,6 +202,16 @@ export const accountPageItems = defineMenuItems({
       },
     },
   },
+  logout: {
+    title: null,
+    items: {
+      logout: {
+        title: 'Log out',
+        icon: ExitIcon,
+        onClick: () => logout(LogoutReason.ManualLogout),
```

Comment:
`logout` from `AuthContext` is the same as the one from `useAuthContext` (latter just sets it in its context 🙈)